### PR TITLE
fix(send): log to STDERR when updating group membership in AD

### PR DESCRIPTION
**Pull request by metodej**
- Group membership update in AD now logs which members were
added/removed to STDERR as well as to local file.

### CODE REVIEW

**Review by Johaney-s — APPROVED** [review]

File: send/ADConnector.pm [hunk 1]
```diff
@@ -575,8 +575,10 @@ sub add_members_to_entry($$$$) {
 		if ($response) {
 			unless ($response->is_error()) {
 				ldap_log($service_name, "Group members added: " . $ad_entry->dn() . " | \n" . join(",\n", @$_));
+				print STDERR "Group members added: " . $ad_entry->dn() . " | \n" . join(",\n", @$_);
```
Comment (cuadradek) [inline send/ADConnector.pm:578]: I think we should only log errors to STDERR.

File: send/ADConnector.pm [hunk 2]
```diff
@@ -609,8 +611,10 @@ sub remove_members_from_entry($$$$) {
 		if ($response) {
 			unless ($response->is_error()) {
 				ldap_log($service_name, "Group members removed: " . $ad_entry->dn() . " | \n" . join(",\n", @$_));
+				print STDERR "Group members removed: " . $ad_entry->dn() . " | \n" . join(",\n", @$_);
```
Comment (cuadradek) [inline send/ADConnector.pm:614]: Same here.

File: send/ADConnector.pm (hunk 1 above)
Comment (metodej) [reply send/ADConnector.pm]: Yes, that would make more sense, thanks.

File: send/ADConnector.pm (hunk 2 above)
Comment (metodej) [reply send/ADConnector.pm]: Also removed.

**Review by cuadradek — APPROVED** [review]

**Pull request merged**